yamljs/yamljs.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for yamljs 0.2.1
+// Type definitions for yamljs 0.2.3
 // Project: https://github.com/jeremyfa/yaml.js
 // Definitions by: Tim Jonischkat <http://www.tim-jonischkat.de>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
-declare module "yamljs" {
```

Comment:
your definitions only support for browser. `declare module "yamljs"` is required by node.js.
your pull request contains breaking change for node.js users.